bindings/node/lib/index.js

```diff
@@ -12,30 +12,47 @@ function loadDefaultModule() {
 const {
   MongoCryptError,
   MongoCryptCreateEncryptedCollectionError,
-  MongoCryptCreateDataKeyError
+  MongoCryptCreateDataKeyError,
+  MongoCryptAzureKMSRequestError,
+  MongoCryptNetworkTimeoutError
 } = require('./errors');
 
+const { fetchAzureKMSToken } = require('./providers');
+const AZURE_PROSE_TESTING_SYMBOL = Symbol.for('@@mdb.azureKMSRefreshProseTest');
```

Comment:
Do you mean directly import from the node_modules?  It could, but that doesn't seem like good practice.

- that goes against all practice we currently have established in how we test mongodb-client-encryption in the driver
- the current implementation documents clearly and with tests the expected behavior to prevent breakages

Unless you feel strongly, I'm going to leave this as-is

Reply:
I meant like this: `require('mongodb-client-encryption/lib/providers/index');` you don't need to put node_modules in the path

Reply:
That doesn't change my mind. Do you feel strongly about this?

Reply:
I don't feel strongly, this just won't work in the future of our ecosystem, eventually ES modules will reach us, and we can't export symbols (IIUC), but that's a distant future if we wanna stick with this for now.